tools/StaticAnalysis/Exceptions/Az.Monitor/SignatureIssues.csv

```diff
@@ -1,3 +1,9 @@
-"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
-"Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.Alerts.NewAzureRmMetricAlertRuleV2CriteriaCommand","New-AzMetricAlertRuleV2Criteria","1","8100","New-AzMetricAlertRuleV2Criteria Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
-"Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.Alerts.NewAzureRmMetricAlertRuleV2DimensionSelectionCommand","New-AzMetricAlertRuleV2DimensionSelection","1","8100","New-AzMetricAlertRuleV2DimensionSelection Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+AssemblyFileName,ClassName,Target,Severity,ProblemId,Description,Remediation
```

Comment:
@shpimpal it looks like StaticAnalysis is failing because each column entry in this file isn't wrapped in quotes (see the deleted lines for how this should look) -- this happens when the contents are copied over from Excel and not done using a text editor, such as VS Code

Reply:
Corrected